google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/ChangeStreamMutation.java

```diff
@@ -145,15 +160,27 @@ abstract static class Builder {
 
     abstract Builder setSourceClusterId(@Nonnull String sourceClusterId);
 
-    abstract Builder setCommitTimestamp(Instant commitTimestamp);
+    Builder setCommitTime(java.time.Instant commitTimestamp) {
+      return setCommitTimestamp(toThreetenInstant(commitTimestamp));
+    }
+
+    /** This method is obsolete. Use {@link #setCommitTime(java.time.Instant)} instead. */
+    @ObsoleteApi("Use setCommitTime(java.time.Instant) instead")
+    abstract Builder setCommitTimestamp(org.threeten.bp.Instant commitTimestamp);
```

Comment:
I think it would be better to remove the threeten setter, only store it as java.time.Instant, and then reverse the overload of getCommitTime(stamp) above.

The builder is fully internal, we just need to maintain backwards compatibility for the getter

Reply:
Thanks for the catch. This was indeed stated in the tracking sheet. I'll update this as an internal class.